### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/92186df9456736665bb2d95f1488cc9baa413032

### CODE REVIEW

**Comment by amychurchwell** [comment]
@peiying2 this seems to have the correct updates. I am not sure why the other PR was generated... 🤔

File: en/firefox/faq.ftl [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+firefoxs-default-search = { -brand-name-firefox }’s default search engine is { -brand-name-google }, which means you can search the web directly from the address bar. <a href="{ $url }">Learn more about search engine preferences and changing defaults.</a>
+
+does-firefox-have = Does { -brand-name-firefox } have a built-in VPN?
+
+# Variables:
+# $url (url) - link to https://fpn.firefox.com/
+# $url2 (url) - link to https://vpn.mozilla.org/
+
+firefox-does-not = { -brand-name-firefox } does not have a built-in VPN (virtual private network), but there are two products made by { -brand-name-mozilla }/{ -brand-name-firefox } that you can use in addition to the private { -brand-name-firefox-browser } that can protect either your browser (<a href="{ $url }">{ -brand-name-firefox-private-network }</a>) or device (<a href="{ $url2 }">{ -brand-name-mozilla-vpn }</a>) connections on WiFi, as well as your IP address.
```
Comment (peiying2) [inline en/firefox/faq.ftl:87]: Should it be singular in `connection on WiFi` as in *either (singular) ABC or (singular) XYZ*?

File: en/firefox/faq.ftl [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+related-questions-ip = Related questions: does { -brand-name-firefox } hide your IP address
+
+who-owns-firefox = Who owns { -brand-name-firefox }?
+
+# Variables:
+# $url (url) - link to https://foundation.mozilla.org
+# $url2 (url) - link to https://www.mozilla.org/foundation/moco/
+
+firefox-is-made = { -brand-name-firefox } is made by { -brand-name-mozilla-corporation  }, a wholly owned subsidiary of the non-profit <a href="{ $url }">{ -brand-name-mozilla-foundation }</a>, and is guided by the principles of the { -brand-name-mozilla } Manifesto. Learn more about the maker of { -brand-name-firefox } <a href="{ $url2 }">here.</a>
```
Comment (peiying2) [inline en/firefox/faq.ftl:97]: - Change from `non-profit` to `not-for-profit`
- { -brand-name-mozilla-corporation  } - extra space before closing bracket. Not sure if it matters.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@amychurchwell 
- The Quantumn file looks good now, thanks for the fix.
- A few comments here and there in the FAQ
- General consistency issue with hyperlinked text in the FAQ page. Should the punctuation be outside the link? It's half and half throughout the page. I didn't call them out but please go through them and make changes.

File: en/firefox/faq.ftl [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+is-firefox-chromium = Is { -brand-name-firefox } { -brand-name-chromium } based?
+firefox-is-not = { -brand-name-firefox } is not based on { -brand-name-chromium } (the open source browser project at the core of { -brand-name-google } { -brand-name-chrome }). In fact, we’re one of the last major browsers that isn’t. { -brand-name-firefox } runs on our Quantum browser engine built specifically for { -brand-name-firefox }, so we can ensure your data is handled respectfully and kept private.
+
+does-firefox-use = Does { -brand-name-firefox } use { -brand-name-google }?
+
+# Variables:
+# $url (url) - link to https://support.mozilla.org/kb/change-your-default-search-settings-firefox
+
+firefoxs-default-search = { -brand-name-firefox }’s default search engine is { -brand-name-google }, which means you can search the web directly from the address bar. <a href="{ $url }">Learn more about search engine preferences and changing defaults.</a>
```
[the remark this answers is not in the document]
Comment (rraue) [reply en/firefox/faq.ftl:79]: @craigcook  had the idea to switch to "Google is the default search engine in Firefox" - lets go with this

**Comment by amychurchwell** [comment]
pmac told me he is going to work on having the same PR update with changes instead of a new one being generated each time. 🎉

**Comment by peiying2** [comment]
The updates are reflected in #124.

**Pull request closed without merging**